src/main/java/com/openlattice/linking/RealtimeLinkingApi.java

```diff
@@ -49,4 +53,16 @@
 
     @GET( BASE + MATCHED + LINKING_ID_PATH )
     Set<MatchedEntityPair> getMatchedEntitiesForLinkingId( @Path( LINKING_ID ) UUID linkingId );
+
+    /**
+     * Description
+     *
+     * @param entitySetIds A JSON list with existing entitySetIds
```

Comment:
Can you please finish the description? Usually you put there what the function/endpoint is for and in the @return you describe the format/specific things about the returned object.